src/Social/Providers/ConfigProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Packages\Social\Providers;
+
+use function Baka\appPath;
+use Phalcon\Di\DiInterface;
+use Phalcon\Di\ServiceProviderInterface;
+use Phalcon\Config;
+
+class ConfigProvider implements ServiceProviderInterface
+{
+    /**
+     * @param DiInterface $container
+     */
+    public function register(DiInterface $container) : void
+    {
+        $container->setShared(
+            'config',
+            function () {
+
+                /**
+                 * @todo Find a better way to handle unit test file include
+                 */
+                $data = require appPath('vendor/canvas/core/src/Core/config.php');
```

Comment:
@rwhite27 kanvs core doesn't hit have this?

Reply:
file no longer necessary, will remove it.